ai8xize.py

```diff
@@ -2626,7 +2626,9 @@ def main():
     while ll < layers:
         if input_channels[ll] <= 0:
             eprint(f'Must specify `in_channels` for layer {ll}.')
-        if operator[ll] != op.NONE:
+        if operator[ll] != op.NONE and quantization[ll] is not None:
+            if quantization[ll] is None:
```

Comment:
is this 2nd if correct? quantization is not None per previous line if

Reply:
I'll take a look at this. It does not look right.